app/reducers/ProductsReducer.jsx

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+
+const initialState = {
+  products: [{name: 'blah', imageURL: 'whatever', price: 10000}]
+}
+
+// CONSTANTS
+export const LOAD_PRODUCTS = 'LOAD_PRODUCTS'
+
+const getProducts = (res) => {
+  return {
+    type: LOAD_PRODUCTS,
+    payload: res.data
+  }
+}
+
+// ACTIONS
+export const loadProducts = () => {
+  return dispatch => {
+    axios.get('/api/products')
+    .then(res => dispatch(getProducts(res)))
+  }
+}
+
+const productsReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case LOAD_PRODUCTS:
+      console.log('ACTION PAYLOAD IN REDUCER', action.payload)
+      return { ...state, products: action.payload }
+    default: return state
+  }
```

Comment:
I see that you have your new state being created and returned inside of your switch statement. Right now your reducer only does one thing, so it'll work, but I would keep in the habit of making a new state at the top of your reducer and returning your new state at the bottom of your reducer. That way you can add potential cases to your switch statement easily and DRY-ly.